src/utils/clmfates_interfaceMod.F90

```diff
@@ -123,7 +123,7 @@ module CLMFatesInterfaceMod
    use EDCanopyStructureMod  , only : canopy_summarization, update_hlm_dynamics
    use FatesPlantRespPhotosynthMod, only : FatesPlantRespPhotosynthDrive
    use EDAccumulateFluxesMod , only : AccumulateFluxes_ED
-   use EDPhysiologyMod       , only : flux_into_litter_pools
+   use EDPhysiologyMod       , only : FluxIntoLitterPools
```

Comment:
Is this a general name change that is taking place? From all lowercase separated by underscores to mixed case without underscores?

Reply:
In general, I'd like us in FATES to use camel case for all procedures (which I believe is consistent with CTSM).  I suppose it would be better to do a complete audit of the procedures and change them all in one go.

Reply:
If you don't do it in one go, you'll likely be inconsistent. But, at least need to make it clear that your goal is to move to camel case for procedure names.